checks/system/unix.py

```diff
@@ -221,6 +221,31 @@ def check(self, agentConfig):
             return False
 
 
+class File_Handlers(Check):
+
+    def check(self, agentConfig):
+
+        if not Platform.is_linux():
+            return False
+
+        try:
+            with open('/proc/sys/fs/file-nr', 'r') as file_handlers:
```

Comment:
on some systems the parent dir of the `sys/` directory is not `/proc`, could you use the value of `procfs_path` in the agent config if it's there? (unless there's a good reason to always use `/proc`)

See https://github.com/DataDog/dd-agent/blob/5.14.1/checks/system/unix.py#L228 for a similar example